src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -2022,7 +2030,7 @@ public void ReadAsyncCallback(IntPtr key, IntPtr packet, uint error)
                 // synchrnously and then call OnTimeoutSync to force an atomic change of state.
                 if (TimeoutHasExpired)
                 {
-                    OnTimeoutSync(asyncClose: true);
+                    OnTimeoutSync(true);
```

Comment:
I think this is another one I added. I think having the parameter name lets you understand what the `true` value means to the callee without looking at the callee code so it helps readability.

Reply:
Reversed the merge and added the param name to the netcore code.